openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
+oeps: {}
+openedx-release: {parent-repo: edx/edx-analytics-dashboard}
```

Comment:
This means when it is time to tag an Open edX release, this repo will be tagged at whatever commit edx-analytics-dashboard is referencing.